pkgs/applications/video/obs-studio/plugins/obs-backgroundremoval/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, obs-studio
+, onnxruntime
+, opencv
+, git
+}:
+
+stdenv.mkDerivation rec {
+  pname = "obs-backgroundremoval";
+  version = "0.5.16";
+
+  src = fetchFromGitHub {
+    owner = "royshil";
+    repo = "obs-backgroundremoval";
+    rev = "v${version}";
+    hash = "sha256-E+pm/Ma6dZTYlX3DpB49ynTETsRS2TBqgHSCijl/Txc=";
+  };
+
+  nativeBuildInputs = [ cmake git ];
```

```suggestion
  nativeBuildInputs = [ cmake ];
```
git isn't required to build this plugin (and the fetchFromGitHub result doesn't contain any git metadata anyway)